src/w3c/headers.js

```diff
@@ -537,15 +537,14 @@ export function run(conf) {
     const pats = [];
     for (let i = 0, n = conf.wg.length; i < n; i++) {
       pats.push(
-        `a <a href='${conf.wgPatentURI[i]}' rel='disclosure'>` +
-          `public list of any patent disclosures (${conf.wg[i]})</a>`
+        hyperHTML`a <a href='${conf.wgPatentURI[i]}' rel='disclosure'>public list of any patent disclosures (${conf.wg[i]})</a>`
       );
     }
-    conf.wgPatentHTML = joinAnd(pats);
+    conf.wgPatentHTML = htmlJoinAnd(pats);
```

Comment:
Could you also replace `${[conf.wgPatentHTML]}` into `${conf.wgPatentHTML}` in `src/w3c/templates/sotd.js`?